src/scenes/Activities/styles.js

```diff
@@ -0,0 +1,14 @@
+const styles = (theme) => ({
+  activitiesTabWrapper: {
+    background: 'white',
```

Comment:
should we use `theme.palette.background.paper`? or is that specifically for Paper components?